packages/server/package.json

```diff
@@ -108,6 +111,7 @@
     "mysql2": "^2.3.1",
     "node-fetch": "2.6.0",
     "open": "^8.4.0",
+    "openapi-types": "^9.3.1",
```

Comment:
This should be moved to devDependencies so that production builds don't include it, typing is only needed for development (`yarn add --dev openapi-types` and remove from `dependencies`).

Reply:
Great shout! added here https://github.com/Budibase/budibase/pull/3644/commits/9d402d03560f9bbbc4d3106d82aa14904bfba79d